src/mainwindow.cpp

```diff
@@ -434,6 +441,10 @@ void MainWindow::showTranslationWindow()
             if (!ui->autoTranslateCheckBox->isChecked())
                 ui->sourceEdit->setRequestTranlationOnEdit(false);
         });
+    } else if (this->isHidden() && settings.windowMode() == AppSettings::Notification) {
+        // If window mode is notification, then dirictly return
+        m_translateSelectionHotkey->blockSignals(false);
+		return;
```

Comment:
You accidentally used tabs here. Please use spaces :)
```suggestion
        return;
```

Reply:
I will correct it.